packages/wdio-jasmine-framework/src/reporter.js

```diff
@@ -19,6 +19,17 @@ export default class JasmineReporter {
         suite.start = new Date()
 
         this.emit('suite:start', suite)
+
+        const hookBeforeAll = Object.assign({}, suite)
```

Comment:
can be moved to a function that accepts hook type as arg

Reply:
Will do!

Reply:
Just refactored this one as well